locale/zh-cn/docs/guides/anatomy-of-an-http-transaction.md

```diff
@@ -0,0 +1,364 @@
+---
+title: HTTP 传输解析
+layout: docs.hbs
+---
+
+# 一次 HTTP 传输解析
+
+本指南的宗旨将让你对 HTTP 传输处理有一个清晰完整的了解。在不考虑特定编程语言及开发环境
```

Comment:
Splitting a Chinese paragraph to multiple lines would result in extra spaces between characters. Please put them to one line.

Reply:
I'm using Visual Studio Code, and it seems I didn't split them into multiple lines……?
Plz have a view of https://github.com/Maledong/nodejs.org/blob/790f8d2ca03796de5347d5a53737708681b5887a/locale/zh-cn/docs/guides/anatomy-of-an-http-transaction.md.

Reply:
@Maledong it’s clear if you see line numbers ;) And actually you could see extra spaces in sentences, with the link you provided.

Reply:
@Maledong This is a must-fix.